aiida/orm/importexport.py

```diff
@@ -1162,12 +1168,27 @@ def import_data_dj(in_path, user_group=None, ignore_unknown_nodes=False,
             if pks_for_group:
                 # If user specified a group, import all things in it
                 if user_group:
-                    group = user_group[0]
+                    if isinstance(user_group, str):
```

Comment:
I feel all this logic is way too complicated for what it should be doing. Wouldn't it be good enough to just accept only a `Group` instance? If one really wants to import everything in a group, creating or loading it first is not that difficult. It would allow to cleanup the following logic mess a lot

Reply:
Have fixed it so that labels can no longer be passed - only Groups

Reply:
Have fixed it so that labels can no longer be passed - only Groups